src/plugins/analysis/linter/internal/js_linter.py

```diff
@@ -1,46 +1,35 @@
-import logging
+import json
+import subprocess
 from pathlib import Path
+from subprocess import PIPE
 
-from common_helper_process import execute_shell_command
-
-CONFIG_FILE_PATH = Path(Path(__file__).parent, 'config', '.jshintrc')
+CONFIG_FILE_PATH = Path(Path(__file__).parent, 'config', 'eslintrc.js')
 
 
 class JavaScriptLinter:
     '''
-    Wrapper for jshint javascript linter
+    Wrapper for eslint javascript linter
     '''
+
     def do_analysis(self, file_path):
-        linter_output = execute_shell_command('jshint --config={} --verbose {}'.format(CONFIG_FILE_PATH, file_path))
-        return self._parse_linter_output(linter_output)
+        # The linter will have nonzero returncode when a rule matches
+        # pylint: disable=subprocess-run-check
+        output_raw = subprocess.run(
```

Comment:
Am I mistaken or is `helperFunctions.docker.run_docker_container` only capable of mounting one volume?

Reply:
Tbh I think building a whole new docker container just for having files in the right place is a bit too much.

The helper functions does not provide that much benefit, does it? The only benefit I see is that it differentiates between DockerExceptions and timeouts.
`subprocess.run` has timeouts as well. What is does not have is DockerExcpetions.

If you would be fine with it, I'd perfer leaving it as it is right now.